applications/GeoMechanicsApplication/tests/settlement_workflow.py

```diff
@@ -0,0 +1,45 @@
+import os
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+import test_helper
+
+class KratosGeoMechanicsSettlementWorkflow(KratosUnittest.TestCase):
+    """
+    This test class is used to check the settlement workflow test, same as test_settlement_workflow.cpp to
+    make sure the python workflow yields the same results as the c++ workflow.
+    """
+
+    def test_DSettlement_workflow(self):
+        test_name = 'test_settlement_workflow'
+        file_path = test_helper.get_file_path(test_name)
+        test_helper.run_stages(file_path, 4)
+
+        times_to_check = [1.0, 2.0, 3.0, 3.2]
+
+        for i in range(4):
+            result_file_name = os.path.join(file_path, f'test_model_stage{i+1}.post.res')
+            expected_result_file_name = (test_helper.get_file_path(
+                os.path.join(test_name, f'test_model_stage{i+1}.post.orig.res')))
```

Comment:
Expected:
expected_result_file_name = os.path.join(file_path, f'test_model_stage{i+1}.post.res')

very similar to line 20.

Reply:
Good catch, done!